src/main/kotlin/lotto/utils/CalculateUtils.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.utils
+
+object CalculateUtils {
+
+    fun getEarningsRate(earningMoney: Long, investMoney: Int): String {
+        val earningsRate = calculateEarningsRate(earningMoney, investMoney)
+        return String.format("%.1f", earningsRate)
+    }
+
+    private const val RATE_PERCENT = 100
+    private const val EMPTY_INVEST_MONEY = 0
+    private const val ZERO_EARNINGS = 0
+    fun calculateEarningsRate(earningMoney: Long, investMoney: Int): Number {
+        val earningsRate = if (investMoney == EMPTY_INVEST_MONEY) {
+            ZERO_EARNINGS
+        } else {
+            (earningMoney.toDouble() / investMoney) * RATE_PERCENT
+        }
+        return earningsRate
+    }
```

Comment:
```suggestion
    fun calculateEarningsRate(earningMoney: Long, investMoney: Int): Number {
        if (investMoney == EMPTY_INVEST_MONEY) return ZERO_EARNINGS
        return (earningMoney.toDouble() / investMoney) * RATE_PERCENT
    }
```
프로그래밍 요구 사항에서 `else` 사용을 지양하고, `if 조건절`에서 값을 `return`하는 방식을 추천한다고 해서 이런 방식은 어떨지 제안합니다!

Reply:
훨씬 좋은 제안인거 같습니다!